src/main/java/Decode.kt

```diff
@@ -0,0 +1,45 @@
+object Decode {
```

Comment:
Praise: Very nice, clean and understandable code.
Nitpick: Lack of comments (at least an author tag would have been nice)